docs/proposals/signal-api.md

```diff
@@ -0,0 +1,174 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Signal API
+
+This document describes signal API effort in Kubernetes.
+
+## Abstract
+
+The goal is to provide a possibility to send signals to container PID 1 processes. At the beginning, we
```

Comment:
I'm not sure if PID1 is the right phrasing here. Perhaps "the entrypoint application" expresses it better?

I'm specifically thinking about rkt where applications are not necessarily PID 1 and about things like hyper/vm runtimes where I suspect the applications will also not be PID 1.